java/core/src/test/java/sleeper/core/record/process/status/ProcessStartedAndFinishedStatus.java

```diff
@@ -20,6 +20,9 @@
 import java.time.Instant;
 import java.util.Objects;
 
+/**
+ * An implementation of a process started update and a process finished update. Only used in tests.
```

Comment:
This seems to apply it's two updates. It's an update that is a start and a finish at the same time.